CMakeLists.txt

```diff
@@ -96,6 +104,9 @@ endif()
 if ( DISABLE_STRING_VIEW )
   target_compile_definitions( date INTERFACE HAS_STRING_VIEW=0 -DHAS_DEDUCTION_GUIDES=0 )
 endif()
+if ( USE_CUSTOM_TZDB_INSTALL_PATH )
```

Comment:
Shouldn't be this in the 
```
if( BUILD_TZ_LIB )
```
block?

I would put it in this line:
```
     if ( USE_SYSTEM_TZ_DB AND NOT WIN32 AND NOT MANUAL_TZ_DB )
         target_compile_definitions( date-tz PRIVATE PUBLIC USE_OS_TZDB=1 )
     else()
          target_compile_definitions( date-tz PUBLIC INSTALL=${USE_CUSTOM_TZDB_INSTALL_PATH} USE_OS_TZDB=0 )
     endif()
```

Se how I removed the other: ` INSTALL=.` together with `USE_OS_TZDB`, that is IMHO an error, will not be considered

Reply:
Oh yeah, I should move it.